### PR TITLE
add note about Teams aplicability

### PR DESCRIPTION
**Pull request by VLG17**
https://github.com/MicrosoftDocs/OfficeDocs-Enterprise/issues/707

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Enterprise/external-domain-name-system-records.md [hunk 1]
```diff
@@ -75,6 +75,9 @@ Email customers who are using Exchange Federation will also need the additional
 <a name="BKMK_ReqdCore"> </a>
 
 There are specific steps to take when you use  [Office 365 URLs and IP address ranges](https://support.office.com/article/8548a211-3fe7-47cb-abb1-355ea5aa88a2#BKMK_LYO) to make sure your network is configured correctly.
+
+> [!NOTE]
+> These DNS records apply also for Teams, especially in a hybrid Teams and Skype for Business Online scenario, where certain federation issues could arise.
```
Comment (JohanFreelancer9) [inline Enterprise/external-domain-name-system-records.md]:
```suggestion
> These DNS records also apply to Teams, especially in a hybrid Teams and Skype for Business Online scenario, where certain federation issues could arise.
```

**Comment by mypil** [comment]
@VLG17 - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by mypil** [comment]
@kelleyvice-msft - PR has been copy edited and is ready for final review. This is based on issue #707

Thank you.

cc: @kenwith

**Comment by kelleyvice-msft** [comment]
Thanks, all!

**Pull request merged**